platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -791,11 +794,9 @@ private void setIsConsensusTrue(@NonNull final EventImpl event, final long recei
         final List<Instant> times = event.getRecTimes();
 
         // take middle. If there are 2 middle (even length) then use the 2nd (max) of them
-        event.setConsensusTimestamp(times.get(times.size() / 2));
+        event.setPreliminaryConsensusTimestamp(times.get(times.size() / 2));
 
         event.setReachedConsTimestamp(Instant.now()); // used for statistics
-
-        consensusMetrics.consensusReached(event);
     }
```

Comment:
having removed `consensusMetrics`, this method `setIsConsensusTrue` may now be `static`